teachertool/src/components/EditorContainer.tsx

```diff
@@ -0,0 +1,32 @@
+import { MakeCodeFrame } from "./MakecodeFrame";
+import { isLocal, getEditorUrl } from "../utils";
+
+const createIFrameUrl = (): string => {
+    const editorUrl: string = isLocal() ? "http://localhost:3232/index.html#editor" : getEditorUrl((window as any).pxtTargetBundle.appTheme.embedUrl);
```

Comment:
Noting here we need to open the editor in read-only mode. A url param may exist for this.